### PR TITLE
virsh_boot: Update error message

### PR DESCRIPTION
**Pull request by cliping**
The error messages were updated recently, so update them.

Signed-off-by: lcheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 virsh.boot.by_ovmf.negative_test.special_boot_order --vt-connect-uri qemu:///system
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : fba4f5d63f7a87be357c69e3528019d6ae43389c
JOB LOG    : /root/avocado/job-results/job-2021-06-23T03.56-fba4f5d/job.log
 (1/3) type_specific.io-github-autotest-libvirt.virsh.boot.by_ovmf.negative_test.special_boot_order.negative: PASS (6.83 s)
 (2/3) type_specific.io-github-autotest-libvirt.virsh.boot.by_ovmf.negative_test.special_boot_order.character: PASS (6.81 s)
 (3/3) type_specific.io-github-autotest-libvirt.virsh.boot.by_ovmf.negative_test.special_boot_order.zero: PASS (6.90 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 21.31 s


#avocado run --vt-type libvirt --vt-machine-type q35 virsh.boot.by_ovmf.negative_test.invalid_loader_type --vt-connect-uri qemu:///system
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 8301182aec9c938d2c99ea20939b49c846cf9f2f
JOB LOG    : /root/avocado/job-results/job-2021-06-23T03.58-8301182/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.boot.by_ovmf.negative_test.invalid_loader_type: PASS (7.64 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 8.38 s

**Review by Yingshun — APPROVED** [review]
LGTM

File: libvirt/tests/cfg/bios/virsh_boot.cfg [hunk 1]
```diff
@@ -146,7 +146,7 @@
                         - special_boot_order:
                             boot_ref = "order"
                             define_error = "yes"
-                            checkpoint = "error: internal error: incorrect boot order.+expecting positive integer"
+                            checkpoint = "error: internal error: incorrect boot order.+expecting positive integer" || "XML error: Invalid value for attribute"
```
Comment (Yingshun) [inline libvirt/tests/cfg/bios/virsh_boot.cfg:149]:
```suggestion
                            checkpoint = '(error: internal error: incorrect boot order.+expecting positive integer|XML error: Invalid value for attribute)'
```

Comment (cliping) [reply libvirt/tests/cfg/bios/virsh_boot.cfg]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

**Review by chloerh — APPROVED** [review]

**Pull request merged**